examples/handlers/opentelemetry_structlog/app.py

```diff
@@ -0,0 +1,47 @@
+from random import randint
+from flask import Flask, request
+import structlog
+import sys
+sys.path.insert(0, '../../..')
+from handlers.opentelemetry_structlog.src.exporter import StructlogHandler
+from opentelemetry.exporter.otlp.proto.grpc._log_exporter import (
+    OTLPLogExporter,
+)
+from opentelemetry.sdk._logs import LoggerProvider
+from opentelemetry.sdk.resources import Resource
+from opentelemetry._logs import set_logger_provider
+from opentelemetry.sdk._logs.export import BatchLogRecordProcessor
+from opentelemetry.sdk.resources import Resource
+
+logger_provider = LoggerProvider(
+    resource=Resource.create(
+        {
+            "service.name": "shoppingcart",
+            "service.instance.id": "instance-12",
+        }
+    ),
+)
+set_logger_provider(logger_provider)
+
+# Replace the standard logging configuration with Loguru
```

Comment:
I think there is a typo

`Replace the standard logging configuration with Loguru` -> `Replace the standard logging configuration with StructLog`